Desserts/Data/Repositories/FavoriteRepository/LocalFavoriteRepository.swift

```diff
@@ -10,18 +10,13 @@ class LocalFavoriteRepository: FavoriteRepository {
     private let favoritesKey = "favorites"
 
     func fetchFavorites() -> [Recipe] {
-        guard let data = UserDefaults.standard.data(forKey: favoritesKey),
-              let favoritesDTO = try? JSONDecoder().decode([RecipeDTO].self, from: data) else {
-            return []
-        }
-        let favorites = favoritesDTO.map{$0.toRecipe()}
-        return favorites
+        let data: FavoritesContainerDTO = Bundle.main.decode(file:"favorites.json")
```

Comment:
I noticed you're using the raw value "favorites" more than once in this files, so I recommend to define it as a constant 🤓

Reply:
Done!